README.md

```diff
@@ -1,62 +1,64 @@
 [![GitHub Actions](https://github.com/python-cmd2/cmd2/workflows/CI/badge.svg)](https://github.com/python-cmd2/cmd2/actions?query=workflow%3ACI)
 [![Azure Build status](https://python-cmd2.visualstudio.com/cmd2/_apis/build/status/python-cmd2.cmd2?branch=master)](https://python-cmd2.visualstudio.com/cmd2/_build/latest?definitionId=1&branch=master)
 [![codecov](https://codecov.io/gh/python-cmd2/cmd2/branch/master/graph/badge.svg)](https://codecov.io/gh/python-cmd2/cmd2)
 [![Documentation Status](https://readthedocs.org/projects/cmd2/badge/?version=latest)](http://cmd2.readthedocs.io/en/latest/?badge=latest)
 <a href="https://discord.gg/RpVG6tk"><img src="https://img.shields.io/badge/chat-on%20discord-7289da.svg" alt="Chat"></a>
 
+
+<p align="center">
+  <a href="#main-features">Main Features</a> •
+  <a href="#installation">Installation</a> •
+  <a href="#tutorials">Tutorials</a> •
+  <a href="#projects-using-cmd2">Projects using cmd2</a> •
+  <a href="#version-two-notes">Version 2.0 Notes</a>
+</p>
+
+[![Screenshot](cmd2.png)](https://youtu.be/DDU_JH6cFsA)
+
 cmd2 is a tool for building interactive command line applications in Python. Its goal is to make it
 quick and easy for developers to build feature-rich and user-friendly interactive command line
 applications.  It provides a simple API which is an extension of Python's built-in
 [cmd](https://docs.python.org/3/library/cmd.html) module.  cmd2 provides a wealth of features on top
 of cmd to make your life easier and eliminates much of the boilerplate code which would be necessary
 when using cmd.
 
-Click on image below to watch a short video demonstrating the capabilities of cmd2:
-[![Screenshot](cmd2.png)](https://youtu.be/DDU_JH6cFsA)
+The developers toolbox
+----------------------
+
+![system schema](https://github.com/python-cmd2/cmd2/blob/README_facelift/.github/images/graph.drawio.png)
 
-Main Features
+
+When creating solutions developers have no shortage of tools in the belt when it comes to rich and smart user interfaces. `cmd2` deftly fills the niche between high [ease of command discovery](https://clig.dev/#ease-of-discovery) and smart workflow automation systems. The above graph is over simplified helps to illustrate when one should reach for cmd2 as the right choice. CLIs have long been the go to pick for those concerned with automation. System administrators have long been duct taping together precarious workflows based on a menagerie of simple command line tools created by strangers on github and the guy down the hall. When CLIs become significantly complex the ease of discoverability tends to fade quickly. Web and GUIs on the other hand are first in class when allowing users to easily discover functionality. The price we pay for beautifully colored displays is it is exceedingly difficult to aggregate many of these applications into larger systems.
```

Comment:
Need to slightly reword `The above graph is over simplified helps to illustrate ...`

Reply:
ya I tend to babble. I'm open to suggestions.